db/seeds.d/07-provisioning_templates.rb

```diff
@@ -12,6 +12,14 @@
   raise "Unable to create template kind: #{format_errors kinds[type]}" if kinds[type].nil? || kinds[type].errors.any?
 end
 
+def is_global_default? input
```

Comment:
Why not put this on the template model itself?
You could then do something like this:

```ruby
if (t = ProvisioningTemplate.find_by_name(input[:name])) && !audit_modified?(ProvisioningTemplate, input[:name])
  next if t.is_global_default?
```